packages/config/config/devices/0x001d/dz6hd.json

```diff
@@ -21,15 +21,27 @@
 			"minValue": 0,
```

Comment:
This and the following parameter should have an explanation of the values in the description:
```
Values 1-127 = seconds; 128-253 = minutes (minus 127)
```